service/controller/v3/resource/flanneld/create.go

```diff
@@ -4,17 +4,11 @@ import (
 	"context"
 
 	"github.com/giantswarm/microerror"
-	"k8s.io/api/extensions/v1beta1"
+	appsv1 "k8s.io/api/apps/v1"
```

Comment:
It turned out we had to go to `apps/v1` with the whole thing.

Reply:
Should we make sure to use `apps` in other operators / operatorkit as well then?